centos2ol.sh

```diff
@@ -1,14 +1,21 @@
 #!/bin/bash
-# Copyright (c) 2020 Oracle and/or its affiliates.
+# Copyright (c) 2020-2021 Oracle and/or its affiliates.
```

Comment:
This is the incorrect syntax for a copyright notice. It should be `Copyright (c) 2020, 2021, Oracle and/or its affiliates.`